### PR TITLE
remove manualInstallPlanApproval flag [MTSRE-693]

### PR DESCRIPTION
**Pull request by venkateshsredhat**
manualInstallPlanApproval flag removal as not implemented on ADO

https://issues.redhat.com/browse/MTSRE-693

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -24,19 +24,19 @@ repos:
   #
   # Built-ins
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
+    rev: v4.5.0
```
Comment (Ankit152) [inline .pre-commit-config.yaml:27]: Have you tested these changes in MT?

Comment (venkateshsredhat) [reply .pre-commit-config.yaml:27]: this based on while running locally but I did run make test and make check . Not sure how to check it in MT

Comment (Ankit152) [reply .pre-commit-config.yaml:27]: ptal: https://gitlab.cee.redhat.com/service/managed-tenants-bundles/-/blob/main/README.md

Comment (venkateshsredhat) [reply .pre-commit-config.yaml:27]: @Ankit152  Could you be precise on how to go about the testing , the read me seems to have lot of information .

Comment (Ankit152) [reply .pre-commit-config.yaml:27]: You could try building the images `locally` in MTB and also test the particular `tasks` present in `MT` repo.

Comment (venkateshsredhat) [reply .pre-commit-config.yaml:27]: @Ankit152 I have updated the ticket https://issues.redhat.com/browse/MTSRE-693 with the output of building images locally in MTB . On running the tasks lets sync .

Comment (Ankit152) [reply .pre-commit-config.yaml:27]: Looks good to me.

**Review by Ankit152 — APPROVED** [review]
LGTM

**Pull request merged**